src/rpm_ostree/cli_status.rs

```diff
@@ -85,37 +86,58 @@ impl Deployment {
             checksum: self.base_revision(),
             version: self.version,
             age_index: None,
+            is_oci: self.container_image_reference.is_some(),
         }
     }
 
     /// Return the deployment base revision.
     pub fn base_revision(&self) -> String {
-        self.base_checksum
+        self.container_image_reference
             .clone()
+            // .map(|pullspec| {
+            //     pullspec
+            //         .strip_prefix("ostree-remote-image:fedora:docker://")
+            //         .map(|s| s.to_string())
+            // })
+            // .flatten()
+            .or(self.base_checksum.clone())
             .unwrap_or_else(|| self.checksum.clone())
     }
+
+    /// return the custom origin fields
+    pub fn custom_origin(&self) -> Option<Vec<String>> {
+        self.custom_origin.clone()
+    }
 }
 
 /// Parse the booted deployment from status object.
 pub fn parse_booted(status: &Status) -> Result<Release> {
     let status = booted_status(status)?;
-    if let Some(img) = status.container_image_reference.as_ref() {
-        let msg = format!("Automatic updates disabled; booted into container image {img}");
-        crate::utils::notify_ready();
-        crate::utils::update_unit_status(&msg);
-        return Err(anyhow::Error::new(SystemInoperable(msg)));
-    }
     Ok(status.into_release())
 }
 
 fn fedora_coreos_stream_from_deployment(deploy: &Deployment) -> Result<String> {
-    let stream = deploy
-        .base_metadata
-        .stream
-        .as_ref()
-        .ok_or_else(|| anyhow!("Missing `fedora-coreos.stream` in commit metadata"))?;
-    ensure!(!stream.is_empty(), "empty stream value");
-    Ok(stream.to_string())
+    if let Some(origin) = deploy.custom_origin.clone() {
+        fedora_coreos_stream_from_custom_origin(&origin[0])
+    } else {
+        let stream = deploy
+            .base_metadata
+            .stream
+            .as_ref()
+            .ok_or_else(|| anyhow!("Missing `fedora-coreos.stream` in commit metadata"))?;
+        ensure!(!stream.is_empty(), "empty stream value");
+        Ok(stream.to_string())
+    }
+}
+
+fn fedora_coreos_stream_from_custom_origin(origin: &String) -> Result<String> {
+    //ostree-remote-image:fedora:registry:quay.io/fedora/fedora-coreos:testing
+    let origin_split: Vec<&str> = origin.split(':').collect();
+    if origin_split.len() != 5 {
+        Err(anyhow!("{origin} is not a valid custom URL"))
+    } else {
+        Ok(origin_split[4].to_string())
```

Comment:
Not sure if Zincati should support other ostree image references ? 

Other URLs schemes are supported by rpm-ostree : https://coreos.github.io/rpm-ostree/container/#url-format-for-ostree-native-containers